llvm/include/llvm/ObjectYAML/DXContainerYAML.h

```diff
@@ -72,6 +73,21 @@ struct ShaderHash {
   std::vector<llvm::yaml::Hex8> Digest;
 };
 
+#define ROOT_ELEMENT_FLAG(Num, Val, Str) bool Val = false;
```

Comment:
I know that this is copying the ShaderFeatureFlags example above, but IMO this would be much clearer if the `#define` was immediately before the `#include`.  I'd be interested to know if there's a reason for this.